src/Avalonia.Controls/TextBlock.cs

```diff
@@ -227,7 +227,7 @@ public TextAlignment TextAlignment
         /// </summary>
         /// <param name="control">The control.</param>
         /// <returns>The font family.</returns>
-        public static string GetFontFamily(Control control)
+        public static FontFamily GetFontFamily(Control control)
```

Comment:
You must also change the SetFontFamily method. 

```
public static void SetFontFamily(Control control, FontFamily value)
        {
            control.SetValue(FontFamilyProperty, value);
        }
```
currently it throws an exception when this is called.